main.cpp

```diff
@@ -1,12 +1,77 @@
 #include <iostream>
+#include <cstdint>
+/*
```

Comment:
co wnosi ten zakomentowany kod ? Albo jest potrzebny, a jak nie to powinien być usunięty a nie w komentarzu